modules/manage/pages/manage-indexes/manage-indexes.adoc

```diff
@@ -0,0 +1,202 @@
+There may also be one or more indicators after the index name, giving further information:
+
++
+image::manage-indexes/index-indicators.png[]
+
+* `partitioned` indicates that the index is {index-partitioning}[partitioned].
+* `replica __n__` indicates that this is an {index-replication}[index replica], where `__n__` is the replica ID.
+* `stale` indicates that the node on which the index or partition is stored is not available.
+
+requests/sec::
+The number of requests per second.
+
+resident ratio::
+The percentage of the data held in memory.
+
+items::
+The number of items currently indexed.
+
+data size::
+The size of indexable data that is maintained for the index or replica.
+
+bucket::
+(Only displayed if viewing the indexes by index, or by server node.)
+The bucket for which the index or replica was created.
+
+node::
+(Only displayed if viewing the indexes by bucket.)
+The server node on which the index or replica is stored, or a list of the nodes on which index partitions are stored.
+
+status::
+The current state of the index service on the node on which this index is stored: *ready*, *pause*, or *warmup*.
++
+The color of the left margin of the index row also reflects the current state of the index.
+For example, the left margin of the index row is green when the index is *ready*, orange when the index is in *warmup*, and so on.
++
+image::manage-indexes/index-margins.png[]
+
+[[expand-index]]
+== Index Administration
+
+To work with an index, click the index row in the indexes list to expand the index row.
+The index row is a toggle -- click the row again to collapse it.
+
+When the index row is expanded, the following extra information is displayed.
+
+image::manage-indexes/index-row-expanded.png[,700,align=left]
+
+Definition::
+The N1QL statement used to define the index.
+
+Storage Mode::
+The {storage-modes}[storage mode] used by the index service on the node on which this index is stored.
+
+Nodes::
+(Only displayed for partitioned indexes.)
+The nodes on which the index partitions are stored, and the number of partitions stored on each node.
+
+Last Scanned::
+The time and data of the last {index-scans}[scan request] received for this index.
+If the index service cannot determine when this index was last scanned, this statistic is not displayed.
+This may be useful for determining whether this index is currently unused.
++
+NOTE: This statistic is persisted to disk every 15 minutes, so it is preserved when the indexer restarts. 
+
+In addition, when the index row is expanded, the *Index Stats* heading is also displayed, along with the *Open in Workbench* and *Drop* buttons.
+
+[[index-stats]]
+=== Show the Index Statistics
+
+To see statistics for the index, click the *Index Stats* heading in the expanded index row.
+
+The *Index Stats* heading is a toggle -- click the heading again to hide the statistics.
+
+For details of the index statistics, refer to {index-stats}[Index Statistics].
+
+[[edit-index]]
+=== Edit the Index
```

Comment:
"Edit the index" can be a little confusing. "Open in workbench" shows the index definition in the query workbench. A user can modify the definition and create a new index using the changed definition but cannot "edit" the existing index. The only way to "edit" an index is using Alter Index command (placement and replica count).